docs/UserGuide.md

```diff
@@ -66,15 +66,27 @@
 * Parameters can be in any order.<br>
   e.g. if the command specifies `n/NAME p/PHONE_NUMBER`, `p/PHONE_NUMBER n/NAME` is also acceptable.
 
-* Extraneous parameters for commands that do not take in parameters (such as `help`, `list`, `exit` and `clear`) will be ignored.<br>
+* Extraneous parameters for commands that do not take in parameters (such as `list`, `exit` and `clear`) will be ignored.<br>
   e.g. if the command specifies `help 123`, it will be interpreted as `help`.
```

Comment:
```suggestion
  e.g. if the command specifies `exit 123`, it will be interpreted as `exit`.
```
I think it is no longer suitable to use `help` as the example here.